Updated for iOS 7 style apps, ARC, Modern Obj-C Syntax

gizmosachin opened the pull request:
I updated the button style to match the appearance of buttons found in the App Store and iTunes Store apps in the iOS 7 betas. I also used Xcode's automatic refactoring tools to update the code to ARC and Modern Objective-C Syntax.

Commit:
Updated for the iOS 7 style, as seen in the iOS 7 beta App Store and iTunes Store apps.

michaelforrest commented:
I'm gonna give this a try. Would be good to see the updated button style in the screenshots in the README with this pull request though.

gizmosachin committed:
Update README.textile

gizmosachin commented:
Thanks Michael. In the commit included with this pull request I did originally update the screenshot in the README with this image: http://i.imgur.com/WhjdTfG.png. Which screenshots are you referring to?

michaelforrest commented:
Yep that's what I was talking about - just didn't see it on this repo's readme. 
BTW I didn't end up using this as it didn't have a podspec to install via CocoaPods and I couldn't untangle the network graph. However, rolling my own hasn't been much fun either!

gizmosachin committed:
Add podspec, fix Sample project, modern syntax

- added Podspec
- changed README to markdown, added sample image to repo

MAConfirmButton
- style fixes
- removed @synthesize statements

Sample Project
- removed all XIB files from sample project
- removed prefix file
- updated Info.plist
- added launch screen XIB to allow sample project to run on any screen size
- now runs correctly in Xcode 6 / iOS 8
- updated build settings to recommended
- added icons, asset catalog

gizmosachin committed:
Fix README

gizmosachin committed:
README changes

gizmosachin committed:
Update project description

gizmosachin committed:
Removed LICENSE file extension

toreolsensan committed:
Remove UIColor-Expanded.

gizmosachin committed:
Merge pull request #1 from toreolsensan/master

Remove UIColor-Expanded.
